### DISCUSSION TITLE
stats for NVIDIA GeForce GTX 1660 SUPER - also looking for suggestions for speeding up

### DISCUSSION THREAD

**Discussion by sputtering** [lllyasviel/Fooocus#1906] [General]
Here are my stats for running some tests using fooocus with an NVIDIA GeForce GTX 1660 SUPER video card. All of these tests are started from run_realistic.bat, Quality, 1 image, same prompt, and the same Seed

1. run_realistic (default sampler is dpmpp_2m_sde_gpu)
    * Generating and saving time: 1107.27 seconds
    * Total time: 1129.26 seconds
2. run_realistic, changed Sampler to dpmpp_2m
    * Generating and saving time: 975.37 seconds
    * Total time: 1022.01 seconds
3. run_realistic, changed Sampler to dpmpp_2m_sde
    * Generating and saving time: 994.63 seconds
    * Total time: 1046.82 seconds

I know it isn't a great card for this stuff but I have to stick with it for at least another month or two.

If anyone has any suggestions for changes that I could test, please let me know. I don't want to sacrifice quality but I would love to cut some minutes off of that render time.

**Comment by 3Diva** [comment]
@sputtering Have you tried the turbo models? That might cut your time a bit as they only require about 5 or so sampling steps. Here's a quick tutorial on how to use turbo models in Fooocus (I'm not sure if this link is allowed or will show up, if not just go to Youtube and search for "How to Use Turbo Models in Fooocus": https://youtu.be/7Fbi17llKsY?si=VXGmb1LDjCRzRCuu

**Reply by mashb1t** [reply]
@sputtering For turbo, you can find a preset in this PR: https://github.com/lllyasviel/Fooocus/pull/1580

You might also try https://github.com/lllyasviel/Fooocus/discussions/1869 / https://github.com/lllyasviel/Fooocus/discussions/1888 for SSD models. They are heavily reduced in size and only take a fraction of the time to generate. Optimized with their LCM LoRAs they're blazing fast with minimal quality tradeoff.

Also see
- https://blog.segmind.com/introducing-segmind-ssd-1b/
- https://huggingface.co/segmind/SSD-1B
- https://huggingface.co/segmind/Segmind-Vega

**Reply by sputtering** [reply]
Thank you for those suggestions!

PR #1580 is an interesting thread, and got me thinking about things:
- Presets and the Performance settings provide a similar function.... Can they be combined? Should I create a new PR to propose this change? --> Oh! Is this what your change in  [#1570 ](url) does?
- You also mention the licensing situation with SDXL Turbo. That also is very interesting. I would love to see that type of note in the GUI or in the log. The problem I assume, is how to tell what the licensing is. In the links you posted in 1580, the model link in CIVIT IA has the licensing note, but the same model on HuggingFace does not.

**Reply by mashb1t** [reply]
IMHO they should not be combined as they serve a different purposes.
This is also why https://github.com/lllyasviel/Fooocus/pull/1570 exists

**Reply by mashb1t** [reply]
You can find more information about SDXL Turbo and its license model here:
- https://stability.ai/core-models
- https://stability.ai/membership

**Comment by xhoxye** [comment]
Try LCM and Turbo.
